www/layouts/partials/ocw_news.html

```diff
@@ -9,51 +9,93 @@
+                                <h3 class="featured-news-heading mobile">
+                                  <a href="{{ $newsItem.link.text }}" class="text-white text-decoration-none">{{ $newsItem.title.text }}</a>
+                                </h1>
+                              </div>
+                            </div>
+                          </a>
+                        </div>
+                      {{ if (or (eq $modulo (sub $itemsInCarousel 1)) (eq $index (sub 5 1))) }}
+                        </div>
+                      {{ end }}
+                    {{ end }}
                   </div>
-                {{ if (or (eq $modulo (sub $itemsInCarousel 1)) (eq $index (sub 4 1))) }}
                 </div>
-                {{ end }}
-              {{ end }}
               </div>
             </div>
           </div>
         {{ end }}
 
-        <div class="d-none d-lg-block">
-          <h2>OCW News</h2>
-          <div class="d-grid items-grid">
-            {{ range $index, $item := (first 4 $items) }}
-              <div class="item item-{{ $index }}"
-                    style="background: linear-gradient(to bottom, transparent, #777777), url({{ $item.image }})">
-                <a href="{{ $item.link.text }}" class="py-2 px-3 text-white text-decoration-none">
-                  {{ $item.title.text }}
-                </a>
+        <h2 class="ocw-news-main-heading">OCW News</h2>
+        <div class="news-container">
+          {{ $firstItem := index $items 0 }}
+            <div class="featured-news d-none d-md-block">
+              <a href="{{ $firstItem.link.text }}">
+                {{ if $firstItem.image }}
+                <img src="{{$firstItem.image}}" alt="">
+                {{ else }}
+                <img src="/static_shared/images/default_news.png" alt="">
+                {{ end }}
+                <div class="news-overlay">
+                  <div class="featured-news-content">
+                    <div class="featured-news-date">{{ time.Format "January 2, 2006" $firstItem.pubDate.text }}</div>
+                    <h3 class="featured-news-heading text-white">
+                      {{ $firstItem.title.text }}
+                    </h3>
+                  </div>
+                </div>
+              </div>
+            </a>
+          <div class="news-sidebar d-none d-md-block">
+            {{ range $index, $item := after 1 (first 5 $items) }}
+            <a href="{{ $item.link.text }}">
+              <div class="news-card">
+                {{ if $item.image }}
+                <img src="{{$item.image}}" alt="">
+                {{ else }}
+                <img src="/static_shared/images/default_news.png" alt="">
+                {{ end }}
+                <div class="news-card-content">
+                  <div class="news-card-title">
+                    <h3 class="">
+                      {{ $item.title.text | truncate 60 }}
```

Comment:
Trim spaces in output
```suggestion
                      {{- $item.title.text | truncate 60 -}}
```